go/golib/sqlutils/sqlutils.go

```diff
@@ -170,20 +170,35 @@ func (this *RowMap) GetTime(key string) time.Time {
 	return time.Time{}
 }
 
+func validateQuery(query string, db *sql.DB) {
+	// dev purposes only. Remove this return to call query validator function.
+	return
+
+	knownDBsMutex.RLock()
+	defer func() {
+		knownDBsMutex.RUnlock()
+	}()
```

Comment:
You can have `defer knownDBsMutex.RUnlock()`

Reply:
done